src/progress.rs

```diff
@@ -145,10 +145,7 @@ impl ProgressDrawTarget {
         match self.kind {
             ProgressDrawTargetKind::Term(ref term, ..) => term.size().1 as usize,
             ProgressDrawTargetKind::Remote(ref state, ..) => state.read().unwrap().width(),
-            ProgressDrawTargetKind::Hidden => {
-                // TODO: Expose `console::term::DEFAULT_WIDTH`?
-                79
-            }
+            ProgressDrawTargetKind::Hidden => 0,
```

Comment:
Could we make this `unreachable!()` or something similar, or is it still called?

Reply:
It was called in `ProgressBar::println` which I have now changed to only format state when it's not hidden. Meaning now it can be safely made unreachable.